### ISSUE TITLE
Is there any `asyncLoaded`?

### ISSUE THREAD

**Issue by Eoksni** [kamijin-fanta/vue-async-data#6]
I would like to have a way to check if the data was loaded. I cannot check for `asyncLoading` to become `false`, because it is `false` initially when data is feeded with default values.

**Comment by kamijin-fanta** [comment]
@Eoksni Thank you feedback.

It does not exist.
I think that it can be solved by specifying a default value like `userNameDefault: undefined`.
then, write the template as like `<span v-if ="userName"> {{userName}} </ span>`.

**Comment by Eoksni** [comment]
True, but its not very convenient when I have several async fields. It then becomes `<span v-if="userName && surName && ...">...</span>` and if boolean values are valid, then I need to check specifically for undefined `<span v-if="userName !== undefined && surName !== undefined && ...">...</span>`
On the other hand, I can encapsulate it in the computed property
```
computed: {
  loaded() {
    return this.userName !== undefined && this.surName !== undefined;
  }
}
```
...
Still, seems a bit errorprone - when adding another async field, I need to remember to put the appropriate check into that computed property.

Some time later I could make a PR for `asyncLoaded` if you don't mind this feature.

**Comment by kamijin-fanta** [comment]
It looks good. welcome PR!
